src/main/java/com/agency04/sbss/pizza/config/PizzaConfig.java

```diff
@@ -1,4 +1,4 @@
-package com.agency04.sbss.pizza;
+package com.agency04.sbss.pizza.config;
 
 import com.agency04.sbss.pizza.service.PizzeriaService;
 import com.agency04.sbss.pizza.service.PizzeriaZagrebService;
```

Comment:
Da li je potrebna "ComponentScan" anotacija nakon što je uključena SpringBoot autokonfiguracija? "Primary" anotaciju prebaciti u klasu za koju to želimo.